scaleway/resource_instance_security_group_rules.go

```diff
@@ -0,0 +1,138 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"strings"
+)
+
+func resourceScalewayInstanceSecurityGroupRules() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayInstanceSecurityGroupRulesCreate,
+		Read:   resourceScalewayInstanceSecurityGroupRulesRead,
+		Update: resourceScalewayInstanceSecurityGroupRulesUpdate,
+		Delete: resourceScalewayInstanceSecurityGroupRulesDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
A custom importer should:
  - Set the `security_group_id` field
  - Call create